src/ImageSharp/Drawing/Brushes/ImageBrush.cs

```diff
@@ -0,0 +1,108 @@
+﻿// <copyright file="ImageBrush.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Drawing.Brushes
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Numerics;
+    using System.Threading.Tasks;
+    using Processors;
+
+    /// <summary>
+    /// Provides an implementaion of a solid brush for painting with repeating images.
+    /// </summary>
+    public class ImageBrush : ImageBrush<Color, uint>
```

Comment:
no problem, I figured actually just adding something that needed it would be the best explanation  😄